plugins/circleci/src/circleci-config.ts

```diff
@@ -12,7 +12,8 @@ import path from 'path'
 import type { PartialDeep } from 'type-fest'
 import YAML from 'yaml'
 
-const majorOrbVersion = '4'
+const MAJOR_ORB_VERSION = '4'
+const DEFAULT_NODE_VERSION = '16.14-browsers'
```

Comment:
**thought**: Should we look into programatically using the version the repo has defined in their package.json?

Reply:
We should do this as part of https://financialtimes.atlassian.net/browse/CPP-1360